tox.ini

```diff
@@ -2,5 +2,5 @@
 envlist = py26, py27, pypy
 
 [testenv]
-deps = -rrequirements.txt
+deps = -rrequirements/dev.txt
```

Comment:
In `tox`, is this supposed to be `-rrequirements/dev.txt`? Shouldn't it be `-rrequirements/dev.txt`